build/yamls/antrea-eks-node-init.yml

```diff
@@ -64,13 +74,11 @@ spec:
                   echo "Waiting for aws-k8s-agent"
               done
 
-              # copied from https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/charts/nodeinit/templates/daemonset.yaml#L199
-              # Fetch running containers from aws-k8s-agent and kill it
+              # copied from https://github.com/cilium/cilium/blob/v1.8.0/install/kubernetes/cilium/charts/nodeinit/templates/daemonset.yaml
```

Comment:
Do not mean your PR, but should we rewrite the script?

Reply:
The script needs to be working ok with the patch. Actually I need to remove the comment about cilium. The code is different from the cilium script now, since they haven't updated it to account for the API change :P